packages/csv-parser-discount-code/test/main.spec.js

```diff
@@ -0,0 +1,118 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Yes, it got fixed by merging csv-parser-order package to master. Please try to update this branch and check if still needed now.